### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by ricardobranco777**
This PR:
- Disables `insecure=True` in Openstack now that we install SUSE certificates.
- Fix a couple of minor pylint issues.

### CODE REVIEW

File: ocw/lib/openstack.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def client(self) -> None:
         if self.__client is None:
             self.__client = openstack.connect(
                 debug=bool(DEBUG),
-                insecure=True,  # Trust the certificate
+                insecure=False,
```
Comment (asmorodskyi) [inline ocw/lib/openstack.py:28]: I am pretty sure that `insecure=False` is default so you can just drop the line

**Review by asmorodskyi — DISMISSED** [review]
LGTM

File: ocw/lib/openstack.py (hunk 1 above)
Comment (ricardobranco777) [reply ocw/lib/openstack.py]: > I am pretty sure that `insecure=False` is default so you can just drop the line

Fixed.

**Review by asmorodskyi — APPROVED** [review]
LGTM

**Review by pdostal — APPROVED** [review]

**Pull request merged**